src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from 'react';
+import { PeopleContext } from '../../store/peopleContext';
+import { PeopleItem } from '../PeopleItem';
+
+export const PeopleTable = () => {
+  const { people } = useContext(PeopleContext);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map((person, index) => (
+          <PeopleItem key={index} person={person} />
```

Comment:
Consider using a unique identifier from the `person` object, such as `person.id` or `person.slug`, for the `key` prop instead of the array index. This helps React optimize rendering performance.